360Controller/Info.plist

```diff
@@ -4515,15 +4515,35 @@
 			<key>idVendor</key>
 			<integer>1848</integer>
 		</dict>
+		<key>ControllerHost</key>
+		<dict>
+			<key>CFBundleIdentifier</key>
+			<string>com.mice.driver.Xbox360Controller</string>
+			<key>IOCFPlugInTypes</key>
+			<dict>
+				<key>F4545CE5-BF5B-11D6-A4BB-0003933E3E3E</key>
+				<string>360Controller.kext/Contents/PlugIns/Feedback360.plugin</string>
+			</dict>
+			<key>IOClass</key>
+			<string>Xbox360Peripheral</string>
+			<key>IOKitDebug</key>
+			<integer>65535</integer>
+			<key>IOProviderClass</key>
+			<string>IOUSBHostDevice</string>
+			<key>idProduct</key>
+			<integer>654</integer>
+			<key>idVendor</key>
+			<integer>1118</integer>
+		</dict>
```

Comment:
This probably needs to replace all instances of `IOUSBDevice` with `IOUSBHostDevice` instead of creating a new `ControllerHost` dictionary.